x-pack/plugins/threat_intelligence/public/modules/indicators/lib/display_name.test.ts

```diff
@@ -11,47 +11,47 @@ describe('display name generation', () => {
   describe('threatIndicatorNamesScript()', () => {
     it('should generate a valid painless script', () => {
       expect(threatIndicatorNamesScript()).toMatchInlineSnapshot(`
-        "if (doc['threat.indicator.type'].size()!=0 && doc['threat.indicator.type'].value!=null && doc['threat.indicator.type'].value.toLowerCase()=='ipv4-addr') { if (doc['threat.indicator.ip'].size()!=0 && doc['threat.indicator.ip'].value!=null) { return emit(doc['threat.indicator.ip'].value) } }
-        if (doc['threat.indicator.type'].size()!=0 && doc['threat.indicator.type'].value!=null && doc['threat.indicator.type'].value.toLowerCase()=='ipv6-addr') { if (doc['threat.indicator.ip'].size()!=0 && doc['threat.indicator.ip'].value!=null) { return emit(doc['threat.indicator.ip'].value) } }
+        "if (doc['threat.indicator.type'].size()!=0 && doc['threat.indicator.type'].value!=null && doc['threat.indicator.type'].value.toLowerCase()=='ipv4-addr') { if (doc.containsKey('threat.indicator.ip') && !doc['threat.indicator.ip'].empty && doc['threat.indicator.ip'].size()!=0 && doc['threat.indicator.ip'].value!=null) { return emit(doc['threat.indicator.ip'].value) } }
```

Comment:
I think we need the same safety checks for the indicator type, I will add them

Reply:
Crap good call, I missed that. We probably should add an e2e test for that...

Reply:
good idea, I will port the changes to the 8.6.0 PR and add a e2e test

Reply:
I just realized this pr is for 8.5, no need of e2e here! Let me know if you want me to take care of the e2e!